class-wc-calypso-bridge-dotcom-features.php

```diff
@@ -34,15 +34,28 @@ function wc_calypso_bridge_has_ecommerce_features() {
 
 if ( ! function_exists( 'wc_calypso_bridge_is_ecommerce_plan' ) ) {
 	/**
-	 * Returns if a site is an eCommerce plan site or not.
+	 * Returns if a site is an eCommerce paid plan site or not.
 	 *
-	 * @return bool True if the site is an ecommerce site.
+	 * @return bool True if the site is an paid ecommerce site.
```

Comment:
Nit: It would be more correct to be `a paid ecommerce site`
```suggestion
	 * @return bool True if the site is a paid ecommerce site.
```